string message
string second_information
bool next_aruco
---
bool success
int32 moreTargets
float32 x
float32 y
float32 z
float32[] vector
int32[] dim#dim[0]=rows; dim[1]=columns
